parse_tz.c

```diff
@@ -878,6 +878,34 @@ int timelib_timestamp_is_in_dst(timelib_sll ts, timelib_tzinfo *tz)
 	return -1;
 }
 
+void timelib_get_time_zone_offset_info(timelib_sll ts, timelib_tzinfo *tz, int32_t* offset, timelib_sll* transition_time, unsigned int* is_dst)
```

Comment:
This API should (like `timelib_get_time_zone_info`) return `0` for failure and`1` for success when the `timelib_fetch_timezone_offset` call fails, instead of returning with all by-ref values set to `0`.